blocks/headline-block/features/headline/default.jsx

```diff
@@ -8,16 +9,42 @@ const HeadlineHeader = styled.h1`
   font-family: ${(props) => props.primaryFont};
 `;
 
-const Headline = ({ useInjectedFusionContext = useFusionContext }) => {
-  const { globalContent: content, arcSite } = useInjectedFusionContext();
+// presentational component handles only visual
+export const Headline = ({ headlineString, primaryFont }) => (
+  /*
+    if string is default empty from presentational container,
+    then render null
+  */
+  (headlineString !== '') && (
+    <HeadlineHeader
+      className="headline"
+      primaryFont={primaryFont}
+      // dangerouslySetInnerHTML seems to be a pattern for blocks
+      dangerouslySetInnerHTML={{ __html: headlineString }}
+    />
+  ));
 
-  return (
-    !!(content && content.headlines && content.headlines.basic) && (
-      <HeadlineHeader className="headline" dangerouslySetInnerHTML={{ __html: content.headlines.basic }} primaryFont={getThemeStyle(arcSite)['primary-font-family']} />
-    )
-  );
+// container handles data fetching, connection to fusion
+const HeadlineContainer = () => {
+  // get headlines basic
+  const { globalContent } = useFusionContext();
+
+  const { basic: headlineString = '' } = globalContent?.headlines || {};
```

Comment:
Definitely can see some helpers happening here eventually! I think as we standardize things in the container method, we'll see a real pattern emerge around how we call the data (since when will now be happening consistently!)